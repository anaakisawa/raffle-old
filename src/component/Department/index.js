import React, { Component } from 'react';
import Header from '../Header';
import Select from '../Select';
import Button from '../Button';

class Department extends Component {
  render () {
    const options = ['auto','vip','tecnologia'];

    return (
      <div className="Department">
        <Header text="Selecione a sua Ã¡rea"/>
        <Select options={options}/>
        <Button text="Ok"/>
      </div>
    );
  }
}

export default Department;
