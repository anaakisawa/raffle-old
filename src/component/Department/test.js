import React from 'react';
import Department from './index';
import renderer from 'react-test-renderer';

describe('Component - Select', () => {
  test('renders a department selection element', () => {
    const component = renderer.create(
      <Department options/>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});
