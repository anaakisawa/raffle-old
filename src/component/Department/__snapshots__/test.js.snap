// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Select renders a department selection element 1`] = `
<div
  className="Department"
>
  <div
    className="Header"
  >
    Selecione a sua Ã¡rea
  </div>
  <div
    className="custom-select"
  >
    <select>
      <option
        value="auto"
      >
        auto
      </option>
      <option
        value="vip"
      >
        vip
      </option>
      <option
        value="tecnologia"
      >
        tecnologia
      </option>
    </select>
  </div>
  <button
    className="Button"
  >
    Ok
  </button>
</div>
`;
